wpilib/tests/test_robotdrive.py

```diff
@@ -46,36 +34,32 @@ def test_init_two(kw, mc, wpimock, halmock):
             mclass = getattr(wpimock, mc)
         else:
             mclass = wpimock.Talon
-        assert isinstance(drive.rearLeftMotor, mclass)
-        assert isinstance(drive.rearRightMotor, mclass)
-        assert drive.rearLeftMotor.getChannel() == left
-        assert drive.rearRightMotor.getChannel() == right
+        assert isinstance(drive.leftMotor, mclass)
+        assert isinstance(drive.leftMotor, mclass)
+        assert drive.leftMotor.getChannel() == left
+        assert drive.rightMotor.getChannel() == right
         # TODO: test hal.setPWM() outputs
 
-@pytest.mark.parametrize("kw,mc",
-        [(False, None), (False, ""), (False, "Victor"),
-         (True, None), (True, ""), (True, "Victor")])
-def test_init_four(kw, mc, wpimock, halmock):
+@pytest.mark.parametrize("kw",
+        [(False), (False), (False),
+         (True), (True), (True)])
+def test_init_four(kw, wpimock, halmock):
     halmock.getFPGATime.return_value = 1000
     halmock.getLoopTiming.return_value = wpimock.SensorBase.kSystemClockTicksPerMicrosecond
 
-    if mc is None:
-        fleft = MagicMock()
-        rleft = MagicMock()
-        fright = MagicMock()
-        rright = MagicMock()
     else:
```

Comment:
Nice syntax error you have there. (Did you actually try to run the tests before committing?)

Reply:
No I’m in the middle of it rn